src/main/java/com/user/management/data/user/dao/impl/UserDaoImpl.java

```diff
@@ -239,10 +228,9 @@ public String getPasswordByUsername(String username) {
 
     @Override
     public String forgotPassword(String username, String newPassword) {
-        String updateQuery = "UPDATE login SET password=?, date_modified=? WHERE username=?";
 
         try (Connection connection = ConnectionHelper.getConnection();
-             PreparedStatement preparedStatement = connection.prepareStatement(updateQuery)) {
+             PreparedStatement preparedStatement = connection.prepareStatement(FORGOT_PASSWORD_STATEMENT)) {
```

Comment:
fix the try-with-resource. remove preparedstatement from try()

Reply:
okay napo sir